libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -6970,4 +6975,16 @@ MAV_RESULT GCS_MAVLINK::handle_control_high_latency(const mavlink_command_int_t
 }
 #endif // HAL_HIGH_LATENCY2_ENABLED
 
+#if AP_RCPROTOCOL_MAVLINK_RADIO_ENABLED
+void GCS_MAVLINK::handle_radio_rc_channels(const mavlink_message_t &msg)
+{
+    mavlink_radio_rc_channels_t packet;
+    mavlink_msg_radio_rc_channels_decode(&msg, &packet);
+
+#if AP_RCPROTOCOL_ENABLED
```

Comment:
```suggestion
```

... it's fair to say if you get here `AP_RCPROTOTOL_ENABLED` is true.

Reply:
THX. done.